packages/f/fancyapps-ui.json

```
@@ -34,7 +34,8 @@
       {
         "basePath": "dist",
         "files": [
-          "*.@(js|css)"
+          "*.js",
+          "*/*.@(js|css)"
```

```suggestion
          "*/*.@(js|css)",
          "*/l10n/*.js"
```
This will let you include those l10n files